.npmignore

```diff
@@ -1,2 +1,5 @@
-node_modules
-webpack.config.js
+**/node_modules/
+**/package.json
+**/index.js
+**/test/
+!**/dist
```

Comment:
i'll add `eslint eol-last: ["error", "always"]` option in eslint ㅎㅎ

Reply:
🙏 👀